dbbackup/tests/test_connectors.py

```diff
@@ -25,14 +26,30 @@ def test_get_connector(self):
 class BaseDBConnectorTest(TestCase):
     def test_init(self):
         connector = BaseDBConnector()
+        assert connector.database_name == DEFAULT_DB_ALIAS
```

Comment:
Could you use `django.test.TestCase`'s specific methods, like `self.assertEqual` ?